app/api/models/checksums.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel, Field
+
+
+class Checksum(BaseModel):
+    md5: str = Field(max_length=32)
```

Comment:
Does it also have a min length?

Reply:
What could the min length be?

Reply:
I was wondering, if it's an md5 checksum, doesn't it mean that it is always 32 characters?

Reply:
I added min_length=32 so it's going to be strictly 32 characters now.